pkg/apis/config/v1alpha4/types.go

```diff
@@ -277,7 +277,7 @@ type PortMapping struct {
 	HostPort int32 `yaml:"hostPort,omitempty"`
 	// TODO: add protocol (tcp/udp) and port-ranges
```

Comment:
Maybe this comment is target of this PR? :thinking:
like as follows: 
```suggestion
	// TODO: add port-ranges
```